ironfish-rust/src/keys/mod.rs

```diff
@@ -241,7 +241,7 @@ impl SaplingKey {
     ///
     /// This method is only called once, but it's kind of messy, so I pulled it
     /// out of the constructor for easier maintenance.
-    fn hash_viewing_key(
+    pub fn hash_viewing_key(
```

Comment:
You will be able to remove this `pub` once you use the method I mentioned above for retrieveing ivk.